compiler/src/lib.rs

```diff
@@ -26,6 +26,7 @@
 pub mod codegen;
 pub mod prost_codegen;
 mod util;
+mod vendored;
```

Comment:
I think we can use a name that represents the function, and the name vendored may be confused with cargo vendor